modules/library/documents/container.php

```diff
@@ -19,6 +19,7 @@ public static function get_properties() {
 		$properties = parent::get_properties();
 
 		$properties['support_kit'] = true;
+		$properties['show_in_finder'] = false;
```

Comment:
@TzviRabinovitch Why does `section.php` have `$properties['show_in_finder'] = true;` and container should be `false`. What is the difference between those two in this regard?

Reply:
That is a question for @avivu